setup.cfg

```
@@ -47,6 +47,10 @@ test_requires =
 packages = find:
 include_package_data = True
 
+[options.entry_points]
+console_scripts =
+    niworkflows=niworkflows.cli.run:main
```

Seems weird to give a specific tool such primacy of place in the namespace, even if we don't have others yet.

```suggestion
    niworkflows-boldref=niworkflows.cli.boldref:main
```